### PR TITLE
Feature/chart markup

### PR DESCRIPTION
**Pull request by JuusoV**
Outlook after styling:
screenshot-localhost 3000 2015-08-07 09-52-34

### CODE REVIEW

**Comment by JuusoV** [comment]
Still couple things needed to be fixed. WIP tag added

**Comment by JuusoV** [comment]
This can be merged now. @apinf/developers

File: client/views/dashboard/charts/filter/filter.html [hunk 1]
```diff
@@ -1,5 +1,10 @@
 <template name="filterData">
-  <h3>Filter by:</h3>
+
+    <div class="box box-primary">
```
Comment (brylie) [inline client/views/dashboard/charts/filter/filter.html]: Check indentation on this file. There may be a semantic error, indicated by the odd indentation structure.

**Comment by brylie** [comment]
Everything seems good, except the odd indentation structure in filter.html. Update the file indentation and double check the opening/closing tag pairs as well as nesting.

**Comment by JuusoV** [comment]
@brylie how does it look now?

**Pull request merged**